frontend/yarn.lock

```diff
@@ -1567,13 +1567,13 @@
   dependencies:
     prettier "^2.0.5"
 
-"@hotosm/underpass-ui@^0.0.4":
-  version "0.0.4"
-  resolved "https://registry.yarnpkg.com/@hotosm/underpass-ui/-/underpass-ui-0.0.4.tgz#3b44f83e7737d550074e9842e8c8873a7e9af01c"
-  integrity sha512-X8sZgfpWKiuGTq3gNeBgLRAH3z33T906Cvgd2fOSfc/MACdXX6Qd8tJ1byX3qkgmd6J0EJsmbZ/KIC/X7oIwCQ==
+"@hotosm/underpass-ui@^0.1.4":
```

Comment:
seems like the latest underpass-ui is `v0.1.6` but it is locked at `v0.1.4`. can you remove node_modules and yarn install again to get the latest version ?

Reply:
cc @emi420